sparkle/src/styles/global_with_tw_base.css

```diff
@@ -1,3 +1,4 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+@import url("https://use.typekit.net/jnb2umy.css");
```

Comment:
What is this?

Reply:
The Darkmode font used in the UI

Reply:
We probably need to add it to `front` as well no?

Reply:
Yes we do.